docker/proof-server-chart/values.yaml

```diff
@@ -0,0 +1,27 @@
+deployment:
+  proofServer:
+    env:
+      containerPort: "8080"
+      rustLog: "debug"
+      rustMinStack: "16777216"
```

Comment:
Here we may want to document that we have that same value in the Docker image too.